Formula/ffmpeg.rb

```diff
@@ -12,78 +12,28 @@ class Ffmpeg < Formula
     sha256 "2e48a23ff807683a78b1792101e39a39cf6ace0e525afefe7109beb421c19034" => :sierra
   end
 
-  option "with-chromaprint", "Enable the Chromaprint audio fingerprinting library"
-  option "with-fdk-aac", "Enable the Fraunhofer FDK AAC library"
-  option "with-libass", "Enable ASS/SSA subtitle format"
-  option "with-librsvg", "Enable SVG files as inputs via librsvg"
-  option "with-libsoxr", "Enable the soxr resample library"
-  option "with-libssh", "Enable SFTP protocol via libssh"
-  option "with-tesseract", "Enable the tesseract OCR engine"
-  option "with-libvidstab", "Enable vid.stab support for video stabilization"
-  option "with-opencore-amr", "Enable Opencore AMR NR/WB audio format"
-  option "with-openh264", "Enable OpenH264 library"
-  option "with-openjpeg", "Enable JPEG 2000 image format"
-  option "with-openssl", "Enable SSL support"
-  option "with-rtmpdump", "Enable RTMP protocol"
-  option "with-rubberband", "Enable rubberband library"
-  option "with-webp", "Enable using libwebp to encode WEBP images"
-  option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
-  option "with-zimg", "Enable z.lib zimg library"
-  option "with-srt", "Enable SRT library"
-  option "with-libvmaf", "Enable libvmaf scoring library"
-  option "with-aom", "Enable AV1 video codec"
-
-  deprecated_option "with-libtesseract" => "with-tesseract"
-
   depends_on "nasm" => :build
   depends_on "pkg-config" => :build
   depends_on "texi2html" => :build
 
+  depends_on "freetype"
+  depends_on "frei0r"
   depends_on "lame"
+  depends_on "libass"
   depends_on "libvorbis"
   depends_on "libvpx"
+  depends_on "opencore-amr"
   depends_on "opus"
+  depends_on "rtmpdump"
   depends_on "sdl2"
   depends_on "snappy"
+  depends_on "speex"
   depends_on "theora"
   depends_on "x264"
   depends_on "x265"
   depends_on "xvid"
   depends_on "xz"
 
-  depends_on "aom" => :optional
-  depends_on "chromaprint" => :optional
-  depends_on "fdk-aac" => :optional
```

Comment:
It's a shame that the best open-source AAC encoder will no longer be available for use in homebrew-core's ffmpeg.

Reply:
See the comment below:

> These librares are GPL-incompatible, and require ffmpeg be built with the "--enable-nonfree" flag, which produces unredistributable libraries

We cannot build this by default as a result and we have removed all options from formulae. I suggest you maintain your own version in a tap.

Reply:
See my comment here: https://github.com/Homebrew/homebrew-core/pull/36663#issuecomment-460485606 - I think for a formula like ffmpeg - if homebrew-core refuses to support options - the only real option is to maintain it as a tap, unless homebrew is willing to create an official tap for formula that "require" options to actually be useful.

Reply:
@RSully Read my message again and please stop posting in multiple threads just to get your voice heard. You are welcome to maintain `ffmpeg` in your own tap with whatever options you want but it will not be in the Homebrew organisation. Furthermore, `ffmpeg` is clearly useful to many people without your desired set of options.

Reply:
@MikeMcQuaid my comment on the other thread was posted before I found this thread. I did not mean any hostility. As you can imagine I am upset with the removal of options in general, and ffmpeg is just the formula that is the perfect example of their need. 

Would homebrew consider creating an "extras" tap as proposed for formula that benefit the most from options?

Or, if a formula were maintained with options outside of homebrew-core, would homebrew continue to support the homebrew-core version, or would it be removed?